datalad/support/annexrepo.py

```diff
@@ -3199,10 +3199,8 @@ def get_content_annexinfo(
                     # of None/NaN etc.
                     del rec['bytesize']
             info[path] = rec
-            # TODO make annex availability checks optional and move in here
-            if not eval_availability:
-                # not desired, or not annexed
-                continue
+        # TODO make annex availability checks optional and move in here
+        if eval_availability:
             self._mark_content_availability(info)
```

Comment:
ha ha -- took me awhile to understand what is going on because diff is a bit "funny" -- this line used to be part of the loop block (thus not marked with +- here). and here is how the code looked before in the loop body:
```python
            info[path] = rec
            # TODO make annex availability checks optional and move in here
            if not eval_availability:
                # not desired, or not annexed
                continue
            self._mark_content_availability(info)
```
if I get it right, such change makes total sense -- approve!